### ISSUE TITLE
Crash when making a call out

### ISSUE THREAD

**Issue by fabriziomoscon** [twilio/twilio-client-quickstart-android#37]
This is a crash in the library not in the demo app.
Although I am aware of the new Programmable voice SDK, I thought it would be still right to record this exception.

lib version `1.2.16`
device Samsung S6 (SM-G925F)
Android system 6.0.1
App in focus

Context: trying to make a call with a UK number with only SMS capabilities. (I am not sure if this matters)

Details from Fabric.io

> Fatal Exception: java.lang.IllegalArgumentException
> Account cannot be null when making a call
>  Raw Text
> com.twilio.client.impl.CallControlManager.newCall (CallControlManager.java:725)
> com.twilio.client.impl.DeviceImpl.handleEventInvite (DeviceImpl.java:628)
> com.twilio.client.impl.CallControlManager.onIncomingCall (CallControlManager.java:961)

![screen shot 2016-10-29 at 13 28 54](https://cloud.githubusercontent.com/assets/721890/19829529/16de6c0a-9ddc-11e6-8482-6e62d728150d.png)

**Comment by kbagchiGWC** [comment]
@fabriziomoscon - was there any error in the log when you created the device? Seems like the account is `null` for the `device` that is receiving the incoming call. Can you provide a log snippet of creating the device? Please set the log level to DEBUG.

**Comment by fabriziomoscon** [comment]
Hi, this error happened on a customer device. So far I only have Fabric setup to catch crashes. So the above info is really all I have to debug the issue. I can give you the account SID of the customer who had the crash, maybe from that you could see if something was misconfigured:
`AC426e882dddc3ecf45f008206ddbe46d8`

I am moving to use the Programmable Voice SDK, so unfortunately it is not likely I will see this bug again while testing.

**Comment by kbagchiGWC** [comment]
@fabriziomoscon does this happen every time they receive a call or it happened only once? `account` can be null due to an invalid capability token. Unfortunately only the account sid will not be sufficient to debug this issue. 

Moving to Voice SDK is the right approach. If this occurs using Voice SDK, please open a new issue.

**Comment by fabriziomoscon** [comment]
Thank you! I will
